lib/common.js

```diff
@@ -178,7 +186,17 @@ function stringifyRequest(options, body) {
 
   if (port) port = ':' + port;
```

Comment:
Port always has a value, so we can move the ":" into the string we are formatting on Line 191 and remove this line.
Importance: Minor